comms/src/peer_manager/node_id.rs

```diff
@@ -42,75 +42,133 @@ use tari_crypto::tari_utilities::{
 const NODE_ID_ARRAY_SIZE: usize = 13; // 104-bit as per RFC-0151
 type NodeIdArray = [u8; NODE_ID_ARRAY_SIZE];
 
+pub type NodeDistance = HammingDistance; // or XorDistance
```

Comment:
Think we should merge this PR and then make another PR to switch it if/when we have experimented a bit with the HammingDistance metric

```suggestion
pub type NodeDistance = XorDistance; // or HammingDistance
```

Reply:
Change applied.